### PR TITLE
Add more ICEs

### PR DESCRIPTION
**Pull request by JohnTitor**

### CODE REVIEW

File: ices/67848.rs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+trait WithAssoc { type AssocType; }
```
Comment (Alexendoo) [inline ices/67848.rs]: Looks like there's a typo in this filename, rust-lang/rust#67848 is a PR

File: ices/66930.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+rustc --crate-type=lib - << EOF
```
Comment (Alexendoo) [inline ices/66930.sh]: FWIW you can specify `#![crate_type = "lib"]` in a .rs and it'll be tested as such, not a big enough deal to suggest changing it in this PR though

File: ices/67848.rs (hunk 1 above)
Comment (JohnTitor) [reply ices/67848.rs]: Good point! It should be 67844.

File: ices/66930.sh (hunk 2 above)
Comment (JohnTitor) [reply ices/66930.sh]: Yeah, we should track `.rs` files basically, fixed.

**Pull request merged**